nemo/collections/tts/helpers/splines.py

```diff
@@ -0,0 +1,306 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Original Source:
+https://github.com/ndeutschmann/zunis/blob/master/zunis_lib/zunis/models/flows/coupling_cells/piecewise_coupling/piecewise_linear.py
+https://github.com/ndeutschmann/zunis/blob/master/zunis_lib/zunis/models/flows/coupling_cells/piecewise_coupling/piecewise_quadratic.py
+Modifications made to jacobian computation
+License: MIT License
+Copyright 2020 Nicolas Deutschmann
+"""
```

Comment:
We need to attach the original copyright headers to this file and file a SWIPAT. Example here: https://github.com/NVIDIA/NeMo/blob/54e6f6ee688f09810d3e54661275fd5c8718db00/nemo/collections/tts/losses/hifigan_losses.py#L16-L36

Reply:
Can you share the SWIPAT templet for such case. Thanks!

Reply:
fixed copyright header and requsted SWIPAT